Formula/s/sentry-native.rb

```diff
@@ -0,0 +1,34 @@
+class SentryNative < Formula
+  desc "Sentry SDK for C, C++ and native applications"
+  homepage "https://docs.sentry.io/platforms/native/"
+  url "https://github.com/getsentry/sentry-native.git",
+    tag:      "0.7.20",
+    revision: "69569b5eca87b5c2afd1ce5a0df7b01ded85023b"
+  license "MIT"
+
+  depends_on "cmake" => :build
+  uses_from_macos "curl"
+  uses_from_macos "zlib"
+
+  def install
+    system "cmake", "-B", "build", *std_cmake_args
```

Comment:
```suggestion
    system "cmake", "-S", ".", "-B", "build", *std_cmake_args
```
Let's follow the style used by other formulae.